spec/cypress/app_commands/scenarios/admin/schools.rb

```diff
@@ -3,4 +3,7 @@
 school = FactoryBot.create(:school, name: "Include this school", urn: 123_456)
 FactoryBot.create(:user, :induction_coordinator, schools: [school])
 
+Faker::UniqueGenerator.clear
+Faker::Config.random = Random.new(42)
```

Comment:
The most random of numbers

Reply:
![](https://www.random.org/analysis/dilbert.jpg)